boot/zephyr/Kconfig

```diff
@@ -384,6 +384,19 @@ endif
 
 endchoice
 
+config BOOT_BYPASS_KEY_MATCH
```

Comment:
@Vge0rge line 74 has changed but doesn't show in PR, can you check it's ok, now is:
```
config BOOT_PSA_IMG_HASH_ALG_SHA256_DEPENDENCIES
	bool
	default y if BOOT_SOMETHING_USES_SHA256
	select PSA_WANT_ALG_SHA_256 if !PSA_CORE_LITE
	help
	  Dependencies for hashing with SHA256
```